reporting_database_generator/elastic_scripts/12_04_publication_gi.sql

```diff
@@ -1,5 +1,5 @@
 {% set elastic_db = "elastic_production_pgpub_" + macros.ds_format(macros.ds_add(dag_run.data_interval_end | ds, -1), "%Y-%m-%d", "%Y%m%d") %}
-{% set reporting_db = "PatentsView_" + macros.ds_format(macros.ds_add(dag_run.data_interval_end | ds, -1), "%Y-%m-%d", "%Y%m%d") %}
+{% set reporting_db = "pregrant_publications" %}
```

Comment:
Reporting db does not exist for pregrant publications. So depending on context this variable need to be removed or set to the PatentsView_ version